### ISSUE TITLE
on the use of 'dehash'

### ISSUE THREAD

**Issue by roycewilliams** [hermanw/dehash#1]
https://www.techsolvency.com/passwords/dehashing-reversing-decrypting/

**Comment by hermanw** [comment]
You are correct when we are talking about "hash" unconditionally. Thanks for the educational efforts!

However in some conditions, for instance, when people apply "hash" to a limited size set, any hash algorithm is no longer secure enough to protect the original information. I see a lot of misunderstandings on this and create this project to tell people that it is so easy to "dehash" or "crack" such kind of protection mechanism.
